Documentation~/handles.md

```diff
@@ -2,19 +2,9 @@
 
 The Cinemachine Handle toolbar is a group of 3D controls that allow you to manipulate virtual camera parameters visually in the Scene view. You can use the handle tools to interactively adjust the selected object's parameters quickly and efficiently rather than controlling them via the inspector.
 
-## Activating and deactivating the toolbar
+The toolbar automatically appears in the Scene view **toolbar** Overlay when you select a type of camera with an associated handle.
```

Comment:
It's better to say virtual camera instead of camera to not confuse the reader. Camera vs Cinemachine Virtual Camera.
type of **virtual camera** with an associated handle.

Reply:
Cool thanks! Comment addressed. Can you also merge for me as usual. Cheers!